apps/dashboard/src/components/conditions-editor/conditions-editor.tsx

```diff
@@ -58,27 +76,119 @@ function InternalConditionsEditor({
   query,
   onQueryChange,
   saveForm,
+  enhancedVariables,
 }: {
-  fields: Field[];
+  fields: Field[] | EnhancedField[];
```

Comment:
do we need the old type here?
```suggestion
  fields: EnhancedField[];
```